python-backend/app/api/mines/mine/resources/mine.py

```diff
@@ -50,48 +56,77 @@ def get(self, mine_no_or_guid=None):
             items_per_page = request.args.get('per_page', 25, type=int)
             page = request.args.get('page', 1, type=int)
             search_term = request.args.get('search', None, type=str)
-            status_search_term = request.args.get('status', None, type=str)
+            status_filter_term = request.args.get('status', None, type=str)
+            tenure_filter_term = request.args.get('tenure', None, type=str)
+            region_code_filter_term = request.args.get('region_code', None, type=str)
+            major_mine_filter_term = request.args.get('major_mine', None, type=str)
+            tsf_filter_term = request.args.get('tsf', None, type=str)
 
-            #Create a filter on mine status if one is provided
-            if status_search_term:
-                status_search_term_array = status_search_term.split(',')
-                status_filter = MineStatusXref.mine_operation_status_code.in_(status_search_term_array)
-                status_reason_filter = MineStatusXref.mine_operation_status_reason_code.in_(status_search_term_array)
-                status_subreason_filter = MineStatusXref.mine_operation_status_sub_reason_code.in_(
-                    status_search_term_array)
-                all_status_filter = status_filter | status_reason_filter | status_subreason_filter
+            sort_criteria = [{'model': 'MineDetail', 'field': 'mine_name', 'direction': 'asc'}]
+            mines_permit_join_query = MineIdentity.query.join(MineDetail)
+            mines_permit_join_query = apply_sort(mines_permit_join_query, sort_criteria)
+            #TODO: Add 'isActive' column to mine status, details, type, location, and permmits tables to allow filtering
```

Comment:
Could you please update this to be an `active_ind` column. And change the `TODO` to be only about what happens here. Example: `TODO: Leverage active_ind column to filter out historical values, created in MDS-###`

And create a ticket for it, which you can reference in the comment. You may also want to check with @klefler to get more information about which tables will be receiving an `active_ind` column at all. Some may not be. She'll have the best perspective on the plans for the database as a whole.

The other option is to just create the ticket and to not leave a comment at all.